sire/standard-library.md

```diff
@@ -13,19 +13,18 @@ Feel free to skim this section for now and come back to it as needed while you w
 This is a printf, a `console.log`.
 
 ```Sire
-(trk %something)
+; input:
+| trk %hello
+| inc 1
 
-:: prints:
-(_Trace %hello)
+; output:
+%hello ; the logged message
+2      ; the result of inc 1
 ```
 
-```Sire
-= x %hello_world
-| trk [%someMessage x]
-
-;; prints:
-trk=[%someMessage {hello_world}]
-```
+`trk` prints its first argument. Its second argument is a continuation - typically the rest of the program. If we were to give `trk` a Haskelly type signature, it would be `trk : a -> b -> b` and we would say that it prints `a` and returns `b`.  
```

Comment:
```suggestion
`trk` prints its first argument. Its second argument is typically the rest of the function we're currently defining. If we were to give `trk` a Haskelly type signature, it would be `trk : a -> b -> b` and we would say that it prints `a` and returns `b`.
```